dll/win32/shell32/dialogs/filedefext.cpp

```diff
@@ -1342,8 +1405,7 @@ CFileDefExt::CountFolderAndFiles(HWND hwndDlg, LPWSTR pwszBuf, UINT cchBufMax, D
 
             ++m_cFolders;
 
-            StringCchCopyW(pwszFilename, cchFilenameMax, wfd.cFileName);
-            CountFolderAndFiles(hwndDlg, pwszBuf, cchBufMax, ticks);
+            CountFolderAndFiles(hwndDlg, sFileName.GetBuffer(), ticks);
```

Comment:
`GetBuffer()` is wrong here (and in all other instances), this leaks memory,
either use `GetString()` or pass the object directly without calling a function, the cast operator is overloaded so this should work just fine.

Reply:
ok